manifests/v1beta1/components/webhook/webhooks.yaml

```diff
@@ -6,8 +6,8 @@ webhooks:
   - name: validator.experiment.katib.kubeflow.org
     sideEffects: None
     failurePolicy: Ignore
-    # TODO (andreyvelich): Migrate to v1 ?
     admissionReviewVersions:
+      - v1
       - v1beta1
```

Comment:
v1beta1 was deprecated in Kubernetes 1.16.
Since 1.16 is not longer maintained by Kubernetes team, should we keep only v1 ?
WDYT @tenzen-y @gaocegege @johnugeorge ?

Reply:
I think so.

Reply:
I left v1beta1 because of the following in the kubeflow documentation.

https://www.kubeflow.org/docs/distributions/aws/deploy/eks-compatibility/

If it is the policy of the Katib community not to support versions of Kubernetes that are not maintained by Kubernetes team, I am in favor of removing v1beta1.

Reply:
AWS distribution has not been updated for Kubeflow 1.3
I think we can say that Katib is working on Kubernetes >= 1.17
It should be fine for the community, otherwise they can use older version of Katib.

Reply:
It makes sense. I'd like to remove v1beta1 from `manifests/v1beta1/components/webhook/webhooks.yaml`.

In addition, I think it needs to be documented that Katib 0.12 supports K8s => 1.17. in `Readme.md` or changelog. What do you think? @andreyvelich

Reply:
> In addition, I think it needs to be documented that Katib supports K8s => 1.17. 

That's a good point, I will point this during refactoring README: https://github.com/kubeflow/katib/issues/1570